app/wdio-validation/specs/actionInteraction-wdio.e2e-spec.ts

```diff
@@ -0,0 +1,148 @@
+      new Date().toLocaleString()
+    );
+
+    TestIdentification.setTmsLink(
+      "TC0001_WebdriverIO-Validation_Action_OnSelectors"
+    );
+    TestIdentification.setDescription(
+      "Goal: The purpose of this test case is to verify different actions performed on" +
+        "elements of the Systelab Components website on different browsers"
+    );
+    await browserService.navigateToSystelabComponents();
+  });
+
+  it("TC0001-01-WebdriverIO-Validation - Navigate to Systelab Components, find an Input Field, set and get a value", async () => {
+    await showCasePage.waitToBeDisplayed();
+    await ReportUtility.addExpectedResult(
+      "The Systelab Components website is open and active",
+      async () => {
+        AssertionUtility.expectContains(
+          await browser.getTitle(),
+          systelabTitle
+        );
+      }
+    );
+
+    await showCasePage.getTextBoxById().setText(foundByID);
+    await ReportUtility.addExpectedResult(
+      "When setting a new value in the systelab component Input Field the value is set correctly",
+      async () => {
+        AssertionUtility.expectEqual(
+          await showCasePage.getTextBoxById().getText(),
+          foundByID
+        );
+      }
+    );
+  });
+
+  it("TC0001-02-WebdriverIO-Validation - Navigate to Systelab Components website, find an Input Field, set and clear a value", async () => {
+    await showCasePage.waitToBeDisplayed();
+    await ReportUtility.addExpectedResult(
+      "The Systelab Components website is open and active",
+      async () => {
+        AssertionUtility.expectContains(
+          await browser.getTitle(),
+          systelabTitle
+        );
+      }
+    );
+    await showCasePage.getTextBoxById().setText(foundByID);
+    await ReportUtility.addExpectedResult(
+      "When setting a new value in the systelab component Input Field the value is set correctly",
+      async () => {
+        AssertionUtility.expectEqual(
+          await showCasePage.getTextBoxById().getText(),
+          foundByID
+        );
+      }
+    );
+    await showCasePage.getTextBoxById().clear();
+    await ReportUtility.addExpectedResult(
+      "When clear the value in the systelab component Input Field the value is clear correctly",
+      async () => {
+        AssertionUtility.expectEqual(
+          await showCasePage.getTextBoxById().getText(),
+          empty
+        );
+      }
+    );
+  });
+
+  it("TC0001-03-WebdriverIO-Validation - Navigate to Systelab Components website and click in a button", async () => {
+    await showCasePage.waitToBeDisplayed();
+    await ReportUtility.addExpectedResult(
+      "The Systelab Components website is open and active",
+      async () => {
+        AssertionUtility.expectContains(
+          await browser.getTitle(),
+          systelabTitle
+        );
+      }
+    );
+    await showCasePage.getIconQuestionButton().click();
+    await modalPage.waitToBeDisplayed(DefaultTimeout.SLOW_WAIT);
```

Comment:
In that case, we have to pass the SLOW_WAIT (5000ms) because when the test clicks on a button then the 500ms default time (FAST_WAIT) it wasn't enough. The modal is not displayed yet.